tests/unit/TodoList1.spec.ts

```diff
@@ -1766,20 +1776,21 @@ describe('TodoList - Click Tests', () => {
     wrapper.destroy()
   })
 
-  it('redirects to Pay URL when \'Retry Payment\' is clicked', async () => {
+  xit('redirects to Pay URL when \'Retry Payment\' is clicked', async () => {
```

Comment:
I'm not sure why, but many of the tests that has line 1824 across multiple files are failing. I would love some guidance on how I might fix the tests (if fixable). 

The line that's failing is:
```javascript
expect(window.location.assign).toHaveBeenCalledWith(payURL + '?accountid=' + accountId)
```
This line is found in other files so I also skipped the tests there. Not all of them though, since it does not always fail.

Reply:
![error](https://github.com/bcgov/business-filings-ui/assets/122301442/3e253880-dac1-4aec-9956-31af5175e27d)
![error 2](https://github.com/bcgov/business-filings-ui/assets/122301442/8846a4bd-4f01-42ec-8f16-6a76cb009d1c)

I believe the line that's causing this error is this:
```javascript
    configurationStore.setTestConfiguration({ configuration: null },
      { key: 'VUE_APP_AUTH_WEB_URL', value: 'https://auth.web.url/' })
```
I can't figure out what `setTestConfiguration` is actually doing and how to fix it. I'll look into more though and try again.

Reply:
The error says that `window.location.assign` wasn't called. Check the code to see what leads to this. Maybe there's some test initialization missing.

Reply:
If this is bogging you down, and you have tested locally and the redirects work correctly, then I propose leaving these "redirect" tests disabled, merging this, and creating a new ticket to fix the tests. Raise this at Standup tomorrow.

Reply:
Is there a new ticket for this?

Reply:
New ticket: https://github.com/bcgov/entity/issues/16392